moesifmiddleware.go

```diff
@@ -75,6 +88,33 @@
 			 multiWriter,
 		 )
 
+		// Disable TransactionId by default
+		disableTransactionId := false
+		// Try to fetch the disableTransactionId from the option
+		if isEnabled, found := moesifOption["disableTransactionId"].(bool); found {
+			disableTransactionId = isEnabled
+		}
+
+		// Add transactionId to the headers
+		if !disableTransactionId {
+			// Try to fetch the transactionId from the header
+			reqTransId := request.Header.Get("X-Moesif-Transaction-Id")
```

Comment:
why not `transactionId = request.Header.Get("X-Moesif-Transaction-Id")`

Reply:
have refactored the code